developer-docs-site/docs/tutorials/testnet-devnet-incentivized-testnet.md

```diff
@@ -0,0 +1,25 @@
+---
+title: "Testnet, Devnet and Incentivized Testnet"
+slug: "testnet-devnet-and-incentivized-testnet"
+---
+
+# Testnet, Devnet and Incentivized Testnet
+
+You can connect to Aptos Blockchain in a few ways. See the below table:
+
+:::tip
+
+Make sure to see the row describing **What not to do**.
+
+:::
+
+|Description |Aptos Devnet | Testnet | Aptos Incentivized Testnet (AIT)|
+|---|---|---|---|
+|**Network runs where**| Runs on Aptos servers.| Runs locally on your computer, entirely independent from devnet. | Runs on Aptos servers.|
+|**Who is responsible for the network**|Managed by Aptos Team. | Deployed, managed and upgraded by you.| Managed by Aptos Team.|
+|**Purpose of the network**|The devnet is built for experimenting with new ideas, improve performance and enhance the user experience.| The testnet is for your development purposes and runs on your local computer.| For executing the Aptos Incentivized Testnet programs for the community.|
```

Comment:
nit, I'd say: "The devnet is built to experiment with new ideas, improve performance and enhance the user experience."